src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+  
+  return days > 2 && days < 7
+    ? total - 20
+    : days > 6
+      ? total - 50
+      : total;
```

Comment:
Prefer using `if` with `return` statements over ternary operators for better readability, especially when the ternary operator is nested. This will make the logic clearer and easier to understand (checklist items 4 and 5).